types/wordpress__components/animate/index.d.ts

```diff
@@ -1,27 +1,34 @@
-import { ComponentType, ReactNode } from 'react';
+import { ComponentType, ReactNode, CSSProperties } from 'react';
 
 declare namespace Animate {
     interface BaseProps {
-        type: 'appear' | 'slide-in';
-        children(props: { className: string }): ReactNode;
+        type: 'appear' | 'slide-in' | 'loading';
+        children:
+            | ReactNode
+            | ((props: { className: '' }) => ReactNode);
     }
 
     interface AppearProps extends BaseProps {
         type: 'appear';
         options?: {
-            origin?: 'top' | 'top left' | 'top right' | 'bottom' | 'bottom left' | 'bottom right';
+            origin?: CSSProperties['transformOrigin'];
```

Comment:
I'm not totally sure about this change. I think it corresponds to my change here but I'd love confirmation of that: https://github.com/WordPress/gutenberg/pull/25754/files#diff-680f5c7d190574aa3514384a3309f40bR19

Reply:
Let's roll back this bit. The full transform origin is much wider and isn't compatible. It can also include arbitrary lengths and is effectively `string`. The following is valid according to the type system:

```ts
import type { Property } from "csstype";

const o: Property.TransformOrigin = "0";
```